toolkit/geo-composables/src/main/java/com/arcgismaps/toolkit/geocomposables/MapComposable.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *  Copyright 2023 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.arcgismaps.toolkit.geocomposables
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+public fun MapComposable() {
```

Comment:
@sorenoid brought up the idea of calling this composable simply `Map()` (and then the 3D equivalent would be `Scene()`). As opposed to `ArcGISMap`, which would have been conflicting with Kotlin's [Map interface](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-map/) if we wouldn't have prefixed it with "ArcGIS", we don't have this issue here since we define a function, not a class.  

Thoughts @shubham7109 @puneet-pdx @rchintapalli @sorenoid @kaushikrw ?

```suggestion
public fun Map() {
```